README.md

```diff
@@ -21,7 +21,80 @@ The goal of this is to:
 If you think you see a bug - write a test-case and let others
 reproduce it on their systems.
 
-This is work in progress. 
+Quickstart
+==========
+
+- Find your localhost owncloud server ip using e.g. `ipconfig`
+- Execute smashbox run over that server e.g. `172.16.12.112:80/octest`
+```
+docker run \
+-e SMASHBOX_URL=172.16.12.112:80/octest \
+-e SMASHBOX_USERNAME=admin \
+-e SMASHBOX_PASSWORD=admin \
+-e SMASHBOX_ACCOUNT_PASSWORD=admin
+-v ~/smashdir:/smashdir \
+owncloud/smashbox lib/test_nplusone.py
+```
+
+Ensure to mount `smashdir` directory to local filesystem to be able to debug test run
+
+- Check run logs
+```
+$ cat ~/smashdir/log-test_nplusone.log | grep error (..warning, critical etc)
+```
+- Check client logs
+```
+$ cat ~/smashdir/test_nplusone/worker0-ocsync.step01.cnt000.log  | grep error (..warning, critical etc)
+```
+- Check sync client directories of workers
+```
+$ ls ~/smashdir/test_nplusone/worker1/
+```
+
+Important integration tests
+===========================
+
+ * [Basic Sync and Conflicts ](lib/test_basicSync.py)
+    - basicSync_filesizeKB from 1kB to 50MB (normal and chunked files sync)
```

Comment:
is this configurable?

Reply:
yes, basic sync can be fonfigurable by flags